src/earth/earth.c

```diff
@@ -546,7 +549,14 @@ earthPreparePaintScreen (CompScreen *s,
 	es->gha = (float)currenttime->tm_hour-(es->tz +
 				(float)currenttime->tm_isdst) +
 				(float)currenttime->tm_min / 60.0f;
-
+				(float)currenttime->tm_sec/3600.0000f;
+	
+	/* Animation */
+            es->rotation += es->rotspeed;
```

Comment:
This is a problem. es->rotspeed is never initialized. See below where es->rotation is initialized.